test/generators/put.js

```diff
@@ -23,10 +23,10 @@ function put_done (total, done) {
 
   return function (_key, record, metadata) {
     return function (err, key, skippy) {
-      switch (err.code) {
-        case status.AEROSPIKE_OK:
+      switch (err) {
```

Comment:
Can replace `switch` statement with an `if .. else` statement since we only check for the presence of the error.